cmd/iamctl/internal/examplecmd/exampleservercmd/server.go

```diff
@@ -81,7 +83,15 @@ func (g googleUserInfoMemberResolver) ResolveIAMMembersFromGoogleUserInfo(
 }
 
 func runServer(ctx context.Context, server iamexamplev1.FreightServiceServer, address string) error {
-	grpcServer := grpc.NewServer(grpc.UnaryInterceptor(logUnary))
+	grpcServer := grpc.NewServer(
+		grpc.UnaryInterceptor(
+			grpc_middleware.ChainUnaryServer(
+				logUnary,
+				iamauthz.RequireUnaryAuthorization,
+			),
```

Comment:
Can this be replace with standard `grpc.ChainUnaryInterceptor`, instead of using `grpc_middleware`?

Reply:
Oh! Yes, didn't know that was available - nice